com.github.christianrauch.Jahresarbeit-2003.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: com.github.christianrauch.Jahresarbeit-2003
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: /app/ja2003
```

Comment:
command is never an absolute path. Also the executable shoud be in `/app/bin`

Reply:
Placing the executable in bin will break the application since it is searching for resource files starting at the directory of the executable. SDL explicitly mentions this for [`SDL_GetBasePath`](https://wiki.libsdl.org/SDL_GetBasePath).

Reply:
what people do in that case is they put a wrapper script in bin.

Standard on UNIX-like is to have executables in `bin`. Otherwise they can't be found..

Reply:
This is actually fine, and Flatpak will not complain about it.